### DISCUSSION TITLE
Using CDN add-on with authorithy to s3 bucket

### DISCUSSION THREAD

**Discussion by dolgovas** [WeblateOrg/weblate#10401] [Q&A] [answered]
Hello! I have weblate installation via docker-compose
I want to use cdn add-on for automatically upload my translate to s3 aws bucket. But I can't find any documents about it.
My bucket has read for all grants and write only with security key. How I can put my translation to this bucket?

**Comment by nijel** [comment, marked as answer]
Presently, this add-on only supports writing to a file system. I've just tried to clarify this in fb924b2b96.

**Reply by dolgovas** [reply]
thanks for your explanation!
